osu.Game.Tests/Visual/Multiplayer/TestSceneMultiplayerMatchSubScreen.cs

```diff
@@ -74,6 +74,25 @@ public void SetupSteps()
         }
 
         [Test]
+        [FlakyTest]
```

Comment:
Plz remove all of these in this class.

Reply:
Actually looks like it may have failed once more in https://teamcity.ppy.sh/buildConfiguration/Osu_Build/1421?hideProblemsFromDependencies=false&hideTestsFromDependencies=false&expandBuildTestsSection=true